common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3205,6 +3205,11 @@
     },
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_FUNCTION_BY_HIVE_VERSION" : {
+    "message" : [
+      "Hive 2.2 and lower versions don't support getTablesByType. Please use Hive 2.3 or higher version."
```

Comment:
Is the `getTablesByType` only function that is not supported by Hive 2.2 and lower?? Otherwise I feel the "FUNCTION" in the error class name might too generic.

Reply:
I guess I agree.
I can rename to "GET_TABLES_BY_TYPE_UNSUPPORTED_BY_HIVE_VERSION" if it's not too long